src/apiserver/api_server_impl.cc

```diff
@@ -459,25 +471,45 @@ void APIServerImpl::RegisterGetTable() {
                           writer << err.Set("Invalid path");
                           return;
                       }
+                      std::vector<std::string> dbs;
+                      hybridse::sdk::Status status;
+                      auto ok = sql_router_->ShowDB(&dbs, &status);
+                      if (!ok) {
+                          writer << err.Set(status.msg);
+                          return;
+                      }
                       auto db = db_it->second;
+                      bool db_ok = std::find(dbs.begin(), dbs.end(), db) != dbs.end();
+                      if (!db_ok) {
+                          writer << err.Set("DB not found");
+                          return;
+                      }
                       auto table = table_it->second;
                       auto table_info = cluster_sdk_->GetTableInfo(db, table);
-                      writer.StartObject();
-                      writer.Member("code") & 0;
+                      
```

Comment:
`src/apiserver/api_server_impl.cc:489:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]`
plz fix it